doc/install.md

```diff
@@ -238,11 +238,16 @@ import pygmt
 pygmt.show_versions()
 
 fig = pygmt.Figure()
-fig.coast(region="g", frame=True, shorelines=1)
+fig.coast(projection="N10c", region="g", frame=True, land="tan", water="lightblue")
+# Add a semi-transparent text element
+fig.text(position="MC", text="PyGMT", font="60p,Helvetica-Bold,red@75")
 fig.show()
 ```
```

Comment:
It would be great if we can add the image here so that it's more straightforward to see the differences if any. But I'm not sure how can it be done here.

Reply:
Yes, this would definitely be nice! I was also thinking about this while writing the explanation comment. I have to see when I have time to figure out a solution - maybe a table: left the code and right the image?